### PR TITLE
SCRUM-2512 (mk2) Use literature service keyword search

### PR DESCRIPTION
**Pull request by markquintontulloch**

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/services/helpers/references/ReferenceSynchronisationHelper.java [hunk 1]
```diff
@@ -53,9 +53,9 @@ protected LiteratureReference fetchLiteratureServiceReference(String curie) {
 
 		HashMap<String, Object> searchField = new HashMap<>();
 		if (curie.startsWith("AGR")) {
-			searchField.put("curie", searchDetails);
+			searchField.put("curie.keyword", searchDetails);
 		} else {
-			searchField.put("cross_references.curie", searchDetails);
+			searchField.put("cross_references.curie.keyword", searchDetails);
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/services/helpers/references/ReferenceSynchronisationHelper.java]: @markquintontulloch why not just search both fields and save the extra code bellow? cross_references.curie won't match AGR and curie wont match mod references?

Comment (markquintontulloch) [reply src/main/java/org/alliancegenome/curation_api/services/helpers/references/ReferenceSynchronisationHelper.java]: I figured search might be faster if only searching one field - or does it not make a difference?

Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/services/helpers/references/ReferenceSynchronisationHelper.java]: no it doesn't make a difference

Comment (markquintontulloch) [reply src/main/java/org/alliancegenome/curation_api/services/helpers/references/ReferenceSynchronisationHelper.java]: @oblodgett - done

**Review by oblodgett — APPROVED** [review]

**Pull request merged**